nle/nethack/nethack.py

```diff
@@ -11,7 +11,7 @@
 
 DLPATH = os.path.join(os.path.dirname(_pynethack.__file__), "libnethack.so")
 
-# TODO: Consider getting this from C++.
+# TODO: Consider getting this from C++. @EH
```

Comment:
Not sure this TODO I put in is super relevant btw. We could easily expose and use the relevant NetHack macros here, but:

* This is _very_ static data, and
* Reading the numbers as plain numbers has some upsides.

BTW my Google training makes me go for `TODO(ghusername)`, although ideally we'd even to `TODO(#issueNo)`.

Reply:
Sorry I actually just put this in as a reminder to do it for this PR! I wouldnt normally put it in like this - it avoids some of the useful highlighting you can get from some editors

Reply:
so i went ahead and changed this to actually get the values from C++. let me know if you think i should change back.